src/message_interface/response.rs

```diff
@@ -142,6 +145,13 @@ pub enum Response {
     /// [`StopBackgroundSync`](crate::message_interface::Message::StopBackgroundSync),
     /// [`EmitTestEvent`](crate::message_interface::Message::EmitTestEvent),
     Ok(()),
+    ////////////////////////////////
+    //// Participation
+    //// ////////////////////////////
```

Comment:
Why the space :carlos: ?

Reply:
To trigger you :trollface:

Reply:
Very efficient :trollface:

Reply:
cargo +nightly fmt does it

Reply:
What? Makes no sense

Reply:
Maybe we don't put a section anyway, we didn't for ledger_nano 🤷🏻‍♂️

Reply:
Worked with only 2 `//` before the Participation, but removed completely now